madrasthemes/mas-elementor
# #60 added button collapse option

- [ ] added button tab option
- [ ] Fixed center slides issue

To test button tab option :
Step 1 : add unique data id
![image](https://github.com/madrasthemes/mas-elementor/assets/86230293/c57ee2a6-9cb4-4b68-a7c3-deb5682818cf)

Step 2 : add data id in the container id
![image](https://github.com/madrasthemes/mas-elementor/assets/86230293/ca326599-4ea3-4402-8e56-9c9fffceb761)

Step 3 : 
Click the button and check in view page
